server/utils/getOrCreateUserDetails.ts

```diff
@@ -21,8 +21,6 @@ export async function findOrCreateUserDetails(
     given_name: true,
     idp: true,
     issuer: true,
-    organization: true,
-    phone: true,
```

Comment:
Are these causing issues? If not we'd like to collect them if they are provided.

Reply:
not causing issues, just not generally returned. I'm content to leave them if you'd prefer